### PR TITLE
Use `pharmaverseadam` data in safety templates - part 1

### LINKED ISSUE #133: Explore and test remaining safety tests can make use of pharmaverseadam data
Remaining safety domains: Exposure, ECG, VS

- [x] [test-listing_cml01.R
- [x] test-listing_cml02a_gl.R
- [x] test-listing_cml02b_gl.R
- [x] test-listing_egl01.R
- [x] test-listing_exl01.R
- [x] test-table_cmt01.R
- [x] test-table_cmt01a.R
- [x] test-table_cmt01b.R
- [x] test-table_cmt02_pt.R

### PR DESCRIPTION
**Pull request by edelarua**
Closes #133

### CODE REVIEW

**Comment by shajoezhu** [comment]
blocked by https://github.com/insightsengineering/scda.test/pull/159

**Comment by ayogasekaram** [comment]
I have read the CLA Document and I hereby sign the CLA

File: tests/testthat/_snaps/table_lbt03.md [hunk 1]
```diff
@@ -7,10 +7,10 @@
                         (N=86)              (N=84)                (N=84)       
       —————————————————————————————————————————————————————————————————————————
       Baseline                                                                 
-        n                 86                  84                    82         
-        Mean (SD)    17.57 (9.22)       19.20 (10.05)          17.96 (8.72)    
-        Median          15.00               16.00                  17.00       
-        Min - Max    7.00 - 69.00        6.00 - 64.00          5.00 - 70.00    
+        n                 0                   0                      0
```
Comment (shajoezhu) [inline tests/testthat/_snaps/table_lbt03.md:10]: this is not right i think. there is probably a bug in the data.

File: tests/testthat/_snaps/listing_lbl02a_rls.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
                                                                        Study   Last Dose of              from       Reference     Reference     Change     Abnormality
       Treatment          Lab Test (Unit)           Center/Patient ID    Day     Study Drug    Result   Baseline       Range         Range     Dec./Inc.       Flag    
       ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
-       Placebo    Alanine Aminotransferase (U/L)       701/1015         -7          0          27.0        0.0     10.0 -  29.0    6 - 34     40% / 40%               
+       Placebo    Alanine Aminotransferase (U/L)       701/1015         -7          0          27.0         NA     10.0 -  29.0    6 - 34     40% / 40%
```
Comment (shajoezhu) [inline tests/testthat/_snaps/listing_lbl02a_rls.md:14]: hi @khatril , this is Baseline value, should we consider display this as 
1.  na_str = "<No data>"
2.  na_str = " "
3.  na_str = "-"

File: tests/testthat/_snaps/table_lbt03.md (hunk 1 above)
Comment (shajoezhu) [reply tests/testthat/_snaps/table_lbt03.md:10]: this is blocked by https://github.com/insightsengineering/tern/issues/1345

**Review by shajoezhu — CHANGES_REQUESTED** [review]
number changes in lab table and listing is not right. lets check these numbers

File: tests/testthat/_snaps/listing_lbl02a_rls.md (hunk 2 above)
Comment (Melkiades) [reply tests/testthat/_snaps/listing_lbl02a_rls.md:14]: it is the baseline so I think it best "-"

**Comment by shajoezhu** [comment]
lgtm! thanks @edelarua , we will move on for now. and follow up with the lbt03 and lab listing issue in https://github.com/insightsengineering/scda.test/issues/162

**Review by shajoezhu — APPROVED** [review]
approve to unblock merge

**Pull request merged**